stories/Templates/AuthorPage/AuthorPage.jsx

```diff
@@ -11,14 +11,25 @@ import { Footer } from '../../Organism/Footer/Footer';
 import './AuthorPage.scss';
 
 const AuthorPage = ({
-  breadcrumbData, authorSummeryData, contentCardData, footerData,
+  breadcrumbData, authorSummeryData, contentCardData, footerData,  languageswitcherData,
```

Comment:
@Ashish-srij kindly correct text "authorSummeryData" to "authorSummaryData".

Reply:
done